client/blocks/image-editor/index.jsx

```diff
@@ -25,29 +26,31 @@ import {
 import {
 	getImageEditorFileInfo
 } from 'state/ui/editor/image-editor/selectors';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getSite } from 'state/sites/selectors';
+import QuerySites from 'components/data/query-sites';
 
-const MediaModalImageEditor = React.createClass( {
+const ImageEditor = React.createClass( {
 	mixins: [ closeOnEsc( 'onCancel' ) ],
 
-	displayName: 'MediaModalImageEditor',
-
 	propTypes: {
-		site: React.PropTypes.object,
-		items: React.PropTypes.array,
-		selectedIndex: React.PropTypes.number,
-		src: React.PropTypes.string,
-		fileName: React.PropTypes.string,
-		mimeType: React.PropTypes.string,
-		setImageEditorFileInfo: React.PropTypes.func,
-		title: React.PropTypes.string,
-		translate: React.PropTypes.func,
-		onImageEditorClose: React.PropTypes.func,
-		onImageEditorCancel: React.PropTypes.func
+		// Component props
+		media: PropTypes.object,
+		siteId: PropTypes.number,
+		onImageEditorClose: PropTypes.func,
+		onImageEditorCancel: PropTypes.func,
+		additionalClasses: PropTypes.string,
```

Comment:
Hmm, good idea. I saw the `additionalClasses` somewhere but this is much better. Thanks!